jte-kotlin/src/main/java/gg/jte/compiler/kotlin/KotlinCodeGenerator.java

```diff
@@ -73,6 +73,13 @@ public void onParam(String parameter) {
         }
         kotlinCode.append(paramInfo.name).append(':').append(paramInfo.type);
 
+        // If there is a default value that IS NOT a gg.jte.Content, then add it to the
+        // method definition. gg.jte.Content are excluded because the syntax is not supported
+        // by Kotlin.
+        if (paramInfo.defaultValue != null && !paramInfo.defaultValue.startsWith("@`")) {
```

Comment:
The same logic repeats here:

https://github.com/casid/jte/pull/285/files#diff-8bf3a3fda545029327a72c0eb5187d04be7a81260716bb2b04a31eecc3ed2029R20

Which I'm not happy with, but I didn't want to pollute `ParaInfo` with code generation logic. Maybe have it declared somewhere where it can be reused?